src/utils/graphql/queries/getReviewInfo.query.ts

```diff
@@ -0,0 +1,26 @@
+import { gql } from '@apollo/client'
+
+export default gql`
+  query getReviewInfo($reviewerId: Int!, $applicationId: Int) {
+    reviewAssignments(condition: { reviewerId: $reviewerId, applicationId: $applicationId }) {
+      nodes {
+        id
```

Comment:
i think we also want level for display

Reply:
but that can be added in the issue of displaying the review assignments

Reply:
No problem. Added: 5620162

Reply:
we would want level from reviewAssignment since review might not exist